app/services/audit/AuditService.scala

```diff
@@ -69,33 +69,56 @@ class AuditService @Inject()(connector: AuditConnector, appConfig: AppConfig)(im
     connector.sendExtendedEvent(extendedEvent).map(handleResponse(_, auditType))
   }
 
-  private def getAuditTags(transactionName: String, path: String)(implicit hc: HeaderCarrier) =
+  def auditMessageInboxPartialRetrieved(eori: String)(implicit hc: HeaderCarrier): Future[AuditResult] = {
+    val auditType = AuditTypes.NavigateToMessages.toString
+    val extendedEvent = ExtendedDataEvent(
+      auditSource = appConfig.appName,
+      auditType = auditType,
+      detail = detailsForMessageInboxPartialRetrieved(eori),
+      tags = tagsForMessageInboxPartialRetrieved
+    )
+
+    connector.sendExtendedEvent(extendedEvent).map(handleResponse(_, auditType))
+  }
+
+  private def getAuditTags(transactionName: String, path: String)(implicit hc: HeaderCarrier): Map[String, String] =
     AuditExtensions
       .auditHeaderCarrier(hc)
       .toAuditTags(transactionName = s"export-declaration-${transactionName.toLowerCase}", path = s"customs-declare-exports/${path}")
 
-  private def handleResponse(result: AuditResult, auditType: String) = result match {
+  private def handleResponse(result: AuditResult, auditType: String): AuditResult = result match {
     case Success =>
       logger.debug(s"Exports ${auditType} audit successful")
       Success
+
     case Failure(err, _) =>
       logger.warn(s"Exports ${auditType} Audit Error, message: $err")
       Failure(err)
+
     case Disabled =>
       logger.warn(s"Auditing Disabled")
       Disabled
   }
 
-  private def getAuditDetails(userInput: JsObject)(implicit hc: HeaderCarrier) = {
+  private def getAuditDetails(userInput: JsObject)(implicit hc: HeaderCarrier): JsObject = {
     val hcAuditDetails = Json.toJson(AuditExtensions.auditHeaderCarrier(hc).toAuditDetails()).as[JsObject]
     hcAuditDetails.deepMerge(userInput)
   }
+
+  private def detailsForMessageInboxPartialRetrieved(eori: String): JsValue =
+    Json.obj("enrolment" -> "HMRC-CUS-ORG", "eoriNumber" -> eori, "tags" -> Json.obj("notificationType" -> "CDS-EXPORTS"))
```

Comment:
I think the values 'enrolment' and 'notificationType' should be passed into the AuditService from the connector where they are defined and used. Hard coding them here means if someone changes these values in the connector they might not know to change them here also.